server/@types/express/index.d.ts

```diff
@@ -30,6 +30,12 @@ import type { UserDetails } from '../../services/userService'
 
 export default {}
 
+declare module 'express-serve-static-core' {
```

Comment:
I have no strong feelings over this, and I am certainly no typescript or express expert! 🤣 , but is there a reason why we create a new module here with the modified Response interface in it, rather than adding it to the global `Express` namespace towards the bottom of this file?

I'm happy if this is the right way to do it; it's more a question so that I might learn something about typescript/express 😁 👍